src/main/java/frc/robot/commands/arm/MoveArm.java

```diff
@@ -9,14 +9,14 @@
 
 import edu.wpi.first.wpilibj2.command.CommandBase;
 import frc.robot.Constants;
-import frc.robot.subsystems.ArmSubsystem;
+import frc.robot.subsystems.FloorArmSubsystem;
```

Comment:
We don’t want this command to be FloorArm but still the original. It should be unchanged